runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/hotcold/HotColdTemperature.java

```diff
@@ -82,9 +82,9 @@
 	 * @param temperatureSet A set of temperature values to select from
 	 * @param message        A string containing a temperature value
 	 * @return The corresponding enum from the given temperature set.
-	 *         <p>
-	 *         Note that in cases where two temperature values in the given set are equally likely to be the given
-	 *         temperature (say, two temperatures with identical text values), the behavior is undefined.
+	 * <p>
```

Comment:
Please revert these random whitespace changes in your PR.

Reply:
I'm not sure how formatting changes are an issue, I just formatted the file using the xml in the dev docs.